recipes/qimage2ndarray/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: qimage2ndarray
+  version: "1.5.1"
+
+source:
+  fn: qimage2ndarray-1.5.1.tar.gz
+  url: https://pypi.python.org/packages/source/q/qimage2ndarray/qimage2ndarray-1.5.1.tar.gz
+  md5: 73743819ccf6043f237c6b7f63732674
+
+build:
+  number: 0
```

Comment:
It is possible to avoid the need for ``build.sh`` by adding a ``build/script`` section in the meta. In this case, I think it makes some sense to do that (though of course, for more complex builds, it is superior to have a ```build.sh```).

One day I will get around to fixing https://github.com/conda/conda-build/issues/682.